_posts/2020-07-27-solidity-v0.7.0-release-announcement.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.7.0 Release Announcement'
+date: '2020-07-27'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity 0.7.0](https://github.com/ethereum/solidity/releases/tag/v0.7.0) is a major breaking release of the Solidity compiler and language.
+
+This release does not include many features but rather changes that require a backwards-incompatible change in syntax or semantics. For an explanation of the features, please see the [documentation](https://solidity.readthedocs.io/en/latest/070-breaking-changes.html).
+
+Most notably, a cleanup of visibility and state mutability has been performed and several unpopular keywords have been removed. Furthermore, types with mappings in memory are now disallowed and a better clarification of types with regards to shift and exponentiation has been implemented.
+
+It is recommended to upgrade all code still using Solidity v0.x.x and older to Solidity v.0.7.0. The [solidity-upgrade tool](https://solidity.readthedocs.io/en/latest/using-the-compiler.html#solidity-upgrade) can help you to semi-automatically upgrade your contracts to breaking language changes. While `solidity-upgrade` carries out a large part of the work, your contracts will most likely need further manual adjustments. We recommend using a version control system for your files, which helps reviewing and eventually rolling back the changes made.
```

Comment:
I don't think we have to mention the benefits of version control systems :)

Reply:
ok, will remove that part.